storage/src/test/scala/coop/rchain/storage/UniTests.scala

```diff
@@ -916,14 +507,17 @@ object UniTests {
   }
 
   def ArraysEqual(a1: Array[String], a2: Array[String]): Boolean = {
```

Comment:
a.k.a.
```
a1.sameElements(a2)
```

Reply:
scala> val x = Array("1","2")
x: Array[String] = Array(1, 2)

scala> val y = Array("2","1")
y: Array[String] = Array(2, 1)

scala> x == y
res0: Boolean = false

scala> x.sameElements(y)
res1: Boolean = false

scala> ArraysEqual(x,y)
res2: Boolean = true

Reply:
Yeah, go ahead and ignore this comment, since it's addressed more fully in #77